src/app/components/SheriffOvertimeStatus.tsx

```diff
@@ -5,11 +5,8 @@ import {
     Badge
 } from 'react-bootstrap';
 
-export interface OvertimeDetailsProps {
-    
-}
 
-export default class SheriffProfile extends React.Component<OvertimeDetailsProps, any>{
+export default class SheriffOvertimeStatus extends React.PureComponent<any>{
```

Comment:
You should be able to use PureComponent without a type ie:
```ts
class MyClass extends React.PureComponent{
...
```

Reply:
fixed